### DISCUSSION TITLE
Can I disable the bell for a single window?

### DISCUSSION THREAD

**Discussion by meatballs** [kovidgoyal/kitty#3612] [Q&A] [answered]
Is there a way to disable the terminal bell for a single window in a tab (rather than for the whole session)?

**Comment by kovidgoyal** [comment, marked as answer]
No, its all or nothing.

**Reply by meatballs** [reply]
Well, thanks for the (very) rapid response! At least I know I'm not missing something obvious!

(Fantastic piece of work you've done here, by the way).

**Reply by kovidgoyal** [reply]
You're welcome :)
